### PR TITLE
Replace placeholder URLs

### PR DESCRIPTION
**Pull request by waitingallday**
## Motivation and Context

**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-

## Changed

1.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

**Comment by waitingallday** [comment]
I went back and forth a bit about where to put the code - it works well in ripple-global, but I don't think we should be publishing an inline image placeholder with ripple 😅 

I ended up locating it in storybook itself since that's where we need it, and added it to `window` for easier (lazier) DX in the story files. Global context seems bad, but I don't know if it's a risk.

**Review by dylankelly — DISMISSED** [review]
This is awesome - nice one. If visual tests pass I think we can go ahead and merge. @waitingallday can you check the snapshots in Chromatic and approve?

**Comment by waitingallday** [comment]
@dylankelly I missed the search example 😕 Chromatic wins again

**Review by dylankelly — APPROVED** [review]

File: packages/components/Organisms/Markup/Markup.stories.mdx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export const Template = (args, { argTypes }) => ({
...
+                  src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%22150%22%20height%3D%22150%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20150%20150%22%20preserveAspectRatio%3D%22none%22%3E%0A%20%20%20%20%3Cdefs%3E%0A%20%20%20%20%20%20%3Cstyle%20type%3D%22text%2Fcss%22%3E%0A%20%20%20%20%20%20%20%20%23holder%20text%20%7B%0A%20%20%20%20%20%20%20%20%20%20fill%3A%20rgb(150%2C%20150%2C%20150)%3B%0A%20%20%20%20%20%20%20%20%20%20font-family%3A%20Arial%2C%20sans-serif%3B%0A%20%20%20%20%20%20%20%20%20%20font-size%3A%2025px%3B%0A%20%20%20%20%20%20%20%20%20%20font-weight%3A%20400%3B%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%3C%2Fstyle%3E%0A%20%20%20%20%3C%2Fdefs%3E%0A%20%20%20%20%3Cg%20id%3D%22holder%22%3E%0A%20%20%20%20%20%20%3Crect%20width%3D%22100%25%22%20height%3D%22100%25%22%20fill%3D%22rgb(204%2C%20204%2C%20204)%22%3E%3C%2Frect%3E%0A%20%20%20%20%20%20%3Cg%3E%0A%20%20%20%20%20%20%20%20%3Ctext%20text-anchor%3D%22middle%22%20x%3D%2250%25%22%20y%3D%2250%25%22%20dy%3D%22.3em%22%3E150%20x%20150%3C%2Ftext%3E%0A%20%20%20%20%20%20%3C%2Fg%3E%0A%20%20%20%20%3C%2Fg%3E%0A%20%20%3C%2Fsvg%3E"
```
[the remark this answers is not in the document]
Comment (waitingallday) [reply packages/components/Organisms/Markup/Markup.stories.mdx:90]: Oh yeah, I couldn't figure out how to embed the JS object into that chunk of HTML so I just generated a 150x150 image on another page and copied the encoded src in. The MDX format treats javascript pretty strangely sometimes..

[the remark this answers is not in the document]
Comment (waitingallday) [reply packages/components/Organisms/Markup/Markup.stories.mdx:90]: It was using the same render method - this is what it outputs on page

**Pull request merged**